### PR TITLE
Remove StatusBarControl member from IShellPage

### PR DESCRIPTION
**Pull request by d2dyno1**
Cleanups to IShellPage and StatusCenter (@duke7553 for reviewing this)

### CODE REVIEW

File: Files/UserControls/StatusCenter.xaml.cs [hunk 1]
```diff
@@ -14,79 +15,100 @@
 
 namespace Files.UserControls
 {
-    public sealed partial class StatusCenter : UserControl
+    public sealed partial class StatusCenter : UserControl, IStatusCenterActions
     {
-        public static ObservableCollection<StatusBanner> StatusBannersSource { get; set; } = new ObservableCollection<StatusBanner>();
+        #region Public Properties
+
+        public ObservableCollection<StatusBanner> StatusBannersSource { get; set; } = new ObservableCollection<StatusBanner>();
```
Comment (d2dyno1) [inline Files/UserControls/StatusCenter.xaml.cs:22]: No longer static!!

File: Files/UserControls/StatusCenter.xaml.cs [hunk 2]
```diff
@@ -14,79 +16,100 @@
...
-        /// <summary>
-        /// Posts a new banner with expanded height to the Status Center control. This is typically
-        /// used to represent a failure during a prior operation which must be acted upon.
-        /// </summary>
-        /// <param name="title"></param>
-        /// <param name="message"></param>
-        /// <param name="primaryButtonText"></param>
-        /// <param name="cancelButtonText"></param>
-        /// <param name="primaryAction"></param>
-        /// <returns>A StatusBanner object which may be used to automatically remove the banner from UI.</returns>
```
Comment (yaichenbaum) [inline Files/UserControls/StatusCenter.xaml.cs:55]: Can you restore the documentation?

**Review by yair100 — CHANGES_REQUESTED** [review]

File: Files/UserControls/StatusCenter.xaml.cs (hunk 2 above)
Comment (d2dyno1) [reply Files/UserControls/StatusCenter.xaml.cs:55]: @yair100 it's been moved to `IStatusCenterActions` interface :)

Comment (d2dyno1) [reply Files/UserControls/StatusCenter.xaml.cs:55]: image

@yair100

File: Files/UserControls/StatusCenter.xaml.cs (hunk 1 above)
Comment (duke7553) [reply Files/UserControls/StatusCenter.xaml.cs]: @d2dyno1 Yes! Does this by any chance effect the displaying of posted banners across tabs? We want the Status Center to show all operations across all tabs.

Comment (d2dyno1) [reply Files/UserControls/StatusCenter.xaml.cs]: @duke7553 haven't thought about it, I'll check it tomorrow. You can check it now if you have time to do so

Comment (d2dyno1) [reply Files/UserControls/StatusCenter.xaml.cs]: @duke7553 I've reverted it back to static

**Comment by d2dyno1** [comment]
@duke7553 @yair100 ready!

**Review by lukeblevins — APPROVED** [review]
Although I dislike the use of non-documentation region markers in code, this LGTM.

**Pull request merged**